bats/tests/containers/switch-engines.bats

```diff
@@ -0,0 +1,56 @@
+# Test case 20
+
+setup() {
+    load '../helpers/load'
+    RD_CONTAINER_ENGINE=moby
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start container engine' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test "pull nginx" {
+    docker run -d -p 8085:80 --restart=no nginx
+    run docker ps --format '{{json .Image}}'
+    assert_output --partial nginx
+}
+
+@test "switch to containerd" {
+    RD_CONTAINER_ENGINE=containerd
+    run rdctl set --container-engine.name=containerd
+    assert_success
+    wait_for_container_engine
```

Comment:
This code could be factored into a function, as it is used twice more below:

```shell
set_container_engine() {
    RD_CONTAINER_ENGINE=$1
    run rdctl set --container-engine.name="$RD_CONTAINER_NAME"
    assert_success
    wait_for_container_engine
}
```

and then call it just like this:

```suggestion
    set_container_engine containerd
```